patches/chrome-android-java-res-values-styles.xml.patch

```diff
@@ -1,10 +1,10 @@
 diff --git a/chrome/android/java/res/values/styles.xml b/chrome/android/java/res/values/styles.xml
-index 048e9e516ee3b5f0c3b58cc4e0968ab8d02f834e..684b33845c9f65ff288fbfa03c7a55a21d1e3a60 100644
+index 048e9e516ee3b5f0c3b58cc4e0968ab8d02f834e..58c9cc6f39e5837ad0b3ff64a3abc9dd79f31f8f 100644
 --- a/chrome/android/java/res/values/styles.xml
 +++ b/chrome/android/java/res/values/styles.xml
 @@ -972,4 +972,5 @@
      <style name="DarkModeCompatibleVerticalScrolling">
          <item name="android:scrollbarThumbVertical">@color/default_scroll_thumb</item>
      </style>
-+    <style name="BraveSwitchTheme"> <item name="colorControlActivated">#fb542b</item> </style>
++    <style name="BraveSwitchTheme"> <item name="colorControlActivated">#fb542b</item> </style> <style name="BraveDialogTheme" parent="Theme.AppCompat.Light.Dialog.Alert"> <item name="windowNoTitle">true</item> </style>
```

Comment:
continuing to add things here doesn't seem like the right approach to me. Where is this file loaded?

Reply:
can we just create our own `brave/android/java/res/values/styles.xml` like we have for strings?

Reply:
Fixed.